ivy/functional/frontends/torch/pointwise_ops.py

```diff
@@ -450,3 +450,11 @@ def hypot(input, other, *, out=None):
 @to_ivy_arrays_and_back
 def sigmoid(input, *, out=None):
     return ivy.sigmoid(input, out=out)
+
+
+@to_ivy_arrays_and_back
+def lerp(input, end, weight, *, out=None):
+    if out is None:
+        out = ivy.zeros_like(input)
+    out = ivy.multiply_add(ivy.subtract(end, input), weight, ivy.subtract(input, out), out=out)
```

Comment:
most lines are redundant, you can just do `return <operation>` as with the above functions (`out` in the docs practically means the `return`) also look over your implementation, `ivy.multiply_add` doesn't exist